src/main/java/chess/domain/score/PieceScore.java

```diff
@@ -0,0 +1,40 @@
+package chess.domain.score;
+
+import chess.domain.piece.Bishop;
+import chess.domain.piece.King;
+import chess.domain.piece.Knight;
+import chess.domain.piece.Pawn;
+import chess.domain.piece.Piece;
+import chess.domain.piece.Queen;
+import chess.domain.piece.Rook;
+import java.util.Arrays;
+
+public enum PieceScore {
+
+    PAWN(Pawn.class, 1),
+    KNIGHT(Knight.class, 2.5),
+    BISHOP(Bishop.class, 3),
+    ROOK(Rook.class, 5),
+    QUEEN(Queen.class, 9),
+    KING(King.class, 0);
+
+    private final Class<? extends Piece> pieceType;
+    private final double score;
+
+    PieceScore(Class<? extends Piece> pieceType, double score) {
+        this.pieceType = pieceType;
+        this.score = score;
+    }
+
+    public static double addScore(Piece piece) {
+        return Arrays.stream(values())
+                .filter(value -> isSamePiece(piece, value))
+                .findFirst()
+                .map(PieceScore -> PieceScore.score)
+                .orElseThrow(() -> new IllegalArgumentException("잘못된 기물입니다."));
```

Comment:
```suggestion
        return Arrays.stream(values())
                .filter(value -> isSamePiece(piece, value))
                .findFirst()
                .orElseThrow(() -> new IllegalArgumentException("잘못된 기물입니다."))
                .score;
```
map을 사용할수도 있지만 `PieceScore`를 찾고 그 score를 전달한다라는 의미를 잘 전달하려면 이런 형태가 어떨까해요.

Reply:
이런 식으로 사용하니 스트림에도 의도가 잘 담기는 것 같아요..!!!!

반영해보겠습니다!